### DISCUSSION TITLE
Code on custom.js not working using child theme.

### DISCUSSION THREAD

**Discussion by franklin-vidal** [bootscore/bootscore#672] [Q&A]
Hello,

Using the bootScore Child theme, code on js/custom.js not working:
![error](https://github.com/bootscore/bootscore/assets/4541817/5a9fbcbe-3100-4f14-9ad4-c7b04077a004)

In another local installation(using LocalWP) is loading custom.js file.

Thanks for help.

**Comment by crftwrk** [comment]
Hi Franklin,

`custom.js` in child is enqueued in `functions.php` line `27` in the WP default way:


```php
wp_enqueue_script('custom-js', get_stylesheet_directory_uri() . '/js/custom.js', false, '', true)
```

This works fine. But if it does not work on your site, we need a link to debug this. Please share a link to your project.

BTW, have you cleared your cache? Child scripts & styles have no timestamp like the main theme yet.

**Comment by franklin-vidal** [comment]
Hi,

I found the error, in front-page.php file i write:
`<?php echo getrf_stylesheet_directory_uri(); ?>`
instead of:
`<?php echo get_stylesheet_directory_uri(); ?>`

After fix the function name, the custom.js load normally on site.

Thanks for help.
